src/app/services/guess-word.service.ts

```diff
@@ -13,4 +13,8 @@ export class GuessWordService {
   checkWord(word: string): Observable<boolean> {
     return this.http.get<boolean>(this.baseURL + '/words/exist?word=' + word);
   }
+
+  newRound() {
+    return this.http.post(this.baseURL + '/rounds/new-round', '');
```

Comment:
```suggestion
    return this.http.post(this.baseURL + '/rounds/new-round', {});
```

Usually when we call a post request without args, we should pass an empty object instead of empty string